arc/runner/common-values.yaml

```diff
@@ -0,0 +1,32 @@
+githubConfigUrl: "https://github.com/utilitywarehouse"
+
+githubConfigSecret: arc-auth
+
+minRunners: 1
+
+containerMode:
+  type: "kubernetes"
+  kubernetesModeWorkVolumeClaim:
+    accessModes: ["ReadWriteOnce"]
+    storageClassName: "netapp-ontap-san-ext4"
+    resources:
+      requests:
+        storage: 5Gi
+
+template:
+  spec:
+    # https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/troubleshooting-actions-runner-controller-errors#error-access-to-the-path-homerunner_work_tool-is-denied
+    securityContext:
+      fsGroup: 123
+    containers:
+      - name: runner
+        image: ghcr.io/actions/actions-runner:latest
```

Comment:
should we be latest?

Reply:
I think so, the runner program auto-updates itself anyway, so we might as well use the latest image directly. Given that the default behaviour it's to auto-update, I don't expect breaking changes